core/trino-spi/src/main/java/io/trino/spi/ptf/TableFunctionSplitProcessor.java

```diff
@@ -27,9 +29,10 @@
     /**
      * This method processes a split. It is called multiple times until the whole output for the split is produced.
      *
-     * @param split a {@link ConnectorSplit} representing a subtask.
+     * @param split a {@link ConnectorSplit} representing a subtask, or {@code null} if a split has already started to be processed,
```

Comment:
Do we get `null` only if we return `TableFunctionProcessorState.Processed.usedInputAndProduced` previously?
and a non-null when we returned `TableFunctionProcessorState.Processed.produced`?

Reply:
correct